### ISSUE TITLE
ThickGUI

### ISSUE THREAD

**Issue by YasirRWebDesigner** [quickemu-project/quickgui#109]
OS: Arch
DE: Hyprland

![swappy-20230727_124528](https://github.com/quickemu-project/quickgui/assets/32537564/3959a9ae-8bac-4224-9b0c-26d1fbb121c9)

So the GUI looks like this on hyprland and idk why, I remember using it on RegataOS gnome and full screen on that worked just fine here not only the GUI looks blurry and THICK and the buttons are also not clickable well kind of you have to find the right spot to be able to click. The quick solution is to just open something else beside it.

![image](https://github.com/quickemu-project/quickgui/assets/32537564/228af2cc-58e4-4a9d-8c89-0843a7f67cc5)

**Comment by GBCam** [comment]
I can almost guarantee that happens because QuickGui is not [set to float](https://wiki.hyprland.org/Configuring/Window-Rules/) in Hyprland.  

Use your binded key to float a window on Hyprland and you should go back to how it looked originally.

**Comment by YasirRWebDesigner** [comment]
> I can almost guarantee that happens because QuickGui is not [set to float](https://wiki.hyprland.org/Configuring/Window-Rules/) in Hyprland.
> 
> Use your binded key to float a window on Hyprland and you should go back to how it looked originally.

yup but that's still not a fix.
![image](https://github.com/quickemu-project/quickgui/assets/32537564/db3e9ae9-09c8-4769-9a3c-249851fafc28)

**Comment by GBCam** [comment]
I think you're misunderstanding that Hyprland runs off Wayland and your old distro runs off xorg.

**Comment by YasirRWebDesigner** [comment]
> I think you're misunderstanding that Hyprland runs off Wayland and your old distro runs off xorg.

no, I think you're misunderstanding I was using Gnome with Wayland.

**Comment by GBCam** [comment]
Yes, your other browser was a desktop environment that had Wayland and gnome, that is something I missed.

But it still comes down that you're using a tiling windows manager through hyprland now.

The window is being stretched originally, which it shows. When you launch your other window, then it brings that stretching down a little. Because it's halfing that window.

But when you float the window, it's going back to the original stated size of the program. 

By all means try the same in a different tiling manager. Do the same in Gnome Wayland and even KDE 6.
